src/cloud-cli/meltano/cloud/api/config.py

```diff
@@ -126,6 +130,7 @@ def __init__(
         )
         self.default_project_id = default_project_id
         self.default_deployment_name = default_deployment_name
+        self.organizations_defaults = organizations_defaults
```

Comment:
I think it would be good if the naming scheme of `organizations_defaults` and `projects` were consistent, since they both represent the same kind of information (i.e. a map from IDs to defaults). Of the two, I think I'd lean towards `*_defaults` for the name, since `projects` seems to imply that it is an exhaustive list of projects, when in fact it just saves defaults for projects you've actively used.

Reply:
I agree, good call on consistency